.github/workflows/pr.yaml

```diff
@@ -1,4 +1,8 @@
-on: pull_request
+on:
+  pull_request:
+  schedule:
+    # “At 04:44 on Saturday.”
+    - cron: '44 4 * * 6'
```

Comment:
Why the schedule statement in the PR workflow?

Reply:
This is a library without a `composer.lock` file. So we never know the specific versions of the installed dependencies because this lies with the using applications. 

Running this at a weekly schedule gives us a chance to detect if never versions of the dependencies breaks the test suite. 

(Yes, SemVer should prevent this from ever happening, but code be code....)